src/RuleSet.php

```diff
@@ -54,19 +54,24 @@ final class RuleSet implements RuleSetInterface
             'single_line_after_imports' => true,
             'switch_case_semicolon_to_colon' => true,
             'switch_case_space' => true,
-            'visibility_required' => true,
+            'visibility_required' => ['elements' => ['method', 'property']],
         ],
         '@PSR12' => [
             '@PSR2' => true,
-            'binary_operator_spaces' => true,
             'blank_line_after_opening_tag' => true,
+            'braces' => [
+                'allow_single_line_anonymous_class_with_empty_body' => true,
```

Comment:
On my local, this doesn't exists... Currently investigating.

Reply:
```suggestion
                'allow_single_line_closure ' => true,
```

Reply:
Probably because I pushed it more recently.

Reply:
Oooh it only exists in `master `branch, for the future 3.0 version.

Reply:
master is not for 3.0, it's for 2.17

Reply:
There is currently no stable version supporting that property, should we really use it now ?

Reply:
Ok forget what I just said, I just noticed that the original PR is against master, so it's fine. I shut up now :D